pkg/cpi/commonUtils.go

```diff
@@ -22,6 +23,11 @@ type CommonUtils interface {
 	GetBearerToken() (string, error)
 }
 
+//CommonUtils for CPI
+type APIMUtils interface {
```

Comment:
let's not put this in commonUtils. Give APIM it's own file please. Also, I'm not sure it should be cpi.apim? Is there a more encompassing name than cpi where apim can be. Because when we refactor cpi it will be cpi.cpi then...

Reply:
done